apps/challenges/views.py

```diff
@@ -1008,7 +1005,7 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
                             or attribute_type == "checkbox"
                         ):
                             options = attribute.get("options")
-                            if not options or not len(options):
+                            if not options or not options:
```

Comment:
This is a redundant check

Reply:
@Ram81 This check doesn't seem to be introduced by DeepSource. I think we should fix it in our code.